assets/src/scss/pages/post/_article-content.scss

```diff
@@ -58,11 +58,6 @@
     }
   }
 
-  blockquote,
```

Comment:
This is already here
https://github.com/greenpeace/planet4-master-theme/blob/5099dcc5464132874f346ae3e501446c897138f2/assets/src/scss/components/_blockquote.scss#L19-L22
which is loaded on all pages
https://github.com/greenpeace/planet4-master-theme/blob/5099dcc5464132874f346ae3e501446c897138f2/assets/src/scss/style.scss#L30-L30